examples/chip-tool/commands/clusters/DataModelLogger.h

```diff
@@ -194,4 +323,5 @@ class DataModelLogger
     }
 
     static size_t ComputePrefixSize(const std::string label, size_t indent) { return ComputePrefix(label, indent).size(); }
+    static DataModelLoggerJSONDelegate * mJSONDelegate;
```

Comment:
Why is DataModelLogger not a namespace? it seems because all the methods are static, now all the class members are static.

Lets make it a namespace and place this as a static global inside the C++ file if that is the intent.

Reply:
As discussed, I will convert that to a namespace in a followup. Is is not purely related to this particular PR and with a lower priority than moving forward with the `matter_yamltests` integration thing.